modulus/metrics/general/ensemble_metrics.py

```diff
@@ -193,7 +193,7 @@ def update(self, inputs: Tensor, dim: int = 0) -> Tensor:
             DistributedManager.is_initialized() and dist.is_initialized()
```

Comment:
isn't checking against `DistributedManager().distributed` sufficient here?

Reply:
Yes, but I'd prefer leaving this change to a separate PR to update the metrics modules to adopt these changes.